src/App.jsx

```diff
@@ -12,26 +12,37 @@ import SiteNav from "./components/SiteNav";
 import NotFound from "./components/NotFound";
 
 function App() {
-  return (
-    <Fragment>
-      <SiteNav />
 
-      <BrowserRouter>
+  const [state, setState] = useState({
+    isRegisterClicked: false
```

Comment:
Is this being used anywhere? Looks like it's being passed as props to SiteNav, but SiteNav doesn't use any props as far as I can tell.

Reply:
This was tied to the Register button that I eventually deleted from SiteNav because Carl's footer already incorporated the Register link. I will delete it.